css/styles.css

```diff
@@ -0,0 +1,69 @@
+.adjust-nav {
+  background-color: #552b81;
+  color: white;
+  font-size: 20px;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  font-family:
+    "Gill Sans",
+    sans-serif;
+}
+
+.add-background-color-main {
+  background-color: #f0f0f0;
+}
+
+.adjust-img {
+  width: 400px;
+  width: 100%;
```

Comment:
You cannot declare the same property twice in one rule. The second one is the one being used, so delete the top one.